modules/manage/partials/tiered-storage.adoc

```diff
@@ -1250,9 +1250,257 @@ endif::[]
 
 == Disaster recovery
 
-When you create a topic, you can use remote recovery to download the topic data from object storage. You can also use remote recovery to restore a topic that was deleted from a cluster.
+With tiered storage enabled, you can restore your cluster, including its metadata, in the event of failure. This is a simpler and less expensive alternative to active-active replication (for example, using xref:upgrade:migrate/data-migration.adoc[MirrorMaker 2]).
 
-CAUTION: Remote recovery is not intended to be a fully-functional disaster recovery solution. It is only safe when no other clusters are still writing to the specified bucket or container. Recovery does not provide snapshot-style consistency: some partitions in some topics will be more up to date than others. For example, restoring a consumer offsets topic could have offsets that point ahead of the restored data.
+You can take the following steps to restore a failed cluster:
+
+. Start a new cluster 
+. Restore data from a failed source cluster to the new cluster
+. Perform a rolling restart, if necessary
+
+Use this recovery method to restore your application to the latest functional state as quickly as possible. 
+
+CAUTION: Whole cluster restore is not intended to be a fully-functional disaster recovery solution. It does not provide snapshot-style consistency: some partitions in some topics will be more up to date than others. For example, restoring a consumer offsets topic could have offsets that point ahead of the restored data, although it will reflect the latest available offset in the topic data that is being restored. Committed transactions are not guaranteed to be atomic.
+
+=== Start a target cluster
+
+For Redpanda Enterprise, follow the steps to https://docs.redpanda.com/current/deploy/deployment-option/self-hosted/[deploy a new cluster] in a self-hosted environment.
+
+For Redpanda Cloud, follow the steps to xref:deploy:deployment-option/cloud/index.adoc[provision a new cloud cluster]. 
+Cloud account credentials (such as secrets or access keys) are not included in backed up metadata. You must manually configure these credentials for the new cluster.
+
+=== Whole cluster restore
+
+Data from a failed cluster can be restored to a target cluster using an rpk command. 
+
+[,bash]
+----
+rpk cluster storage restore start
```

Comment:
@andrwng Are there required parameters? Does the command need to specify the source and destination clusters?

Also, should the `stop` command be mentioned at any point in this doc? Would that go under the [post-restart section](https://github.com/redpanda-data/docs/pull/118/files#diff-8e053dde88b9dfc92939a73598f0d06dd718f88b36c08f136745e3cf97bcb061R1483) below?

Reply:
>@andrwng Are there required parameters? Does the command need to specify the source and destination clusters?

The main one is `-w`. I haven't implemented support for `--topic-name-pattern` but I think it's a bit simpler to leave it out for now.

@mattschumpert given this is whole cluster restore, should we just remove the topic pattern?

Reply:
@andrwng im not sure why we need the topic pattern with cluster storage recovery.